mir_eval/chord.py

```diff
@@ -1342,6 +1347,136 @@ def sevenths_inv(reference_labels, estimated_labels):
     return comparison_scores
 
 
+def directional_hamming_distance(reference_intervals, estimated_intervals):
+    """Compute the directional hamming distance between reference and
+    estimated intervals as defined by [#harte2010towards]_ and used for MIREX
+    'OverSeg', 'UnderSeg' and 'MeanSeg' measures.
+
+    Examples
```

Comment:
_Technically_, since this isn't a metric function, you don't need an "Examples" section, but it certainly doesn't hurt :)